### DISCUSSION TITLE
Tell me about your favorite Netflix show!!

### DISCUSSION THREAD

**Discussion by mana-makinae** [NAIST-SE/AttractingContributors#488] [General]
My hobby is watching Netflix shows.
What is your favorite Netflix show? Feel free to tell me!!

My fork : https://github.com/mana-makinae/AttractingContributors-favorite-netflix-show

**Comment by tmymzkr** [comment]
I done pull request 
please help me
https://github.com/m-tomoya/AttractingContributors-movie
